### PR TITLE
[Case study] Identity and Yahoo! JAPAN

### LINKED ISSUE #7392: [Payments/Identity] Case Study: Identity at Yahoo JAPAN [3/31/2022]
* Proposed title: Case Study: Identity at Yahoo JAPAN
* Proposed by: agektmr
* Related areas: Payments/Identity
* Content type: Case Study (https://web.dev/handbook/case-study-guidelines/)
* Deadline: 3/31/2022
* Deadline explanation: N/A
* [Draft](https://docs.google.com/document/d/1I-Sk40RL7RG42MBhB0ByFp7HgAaQLoFm6PwDOsgOzv8/edit#)
* Content summary: Yahoo! JAPAN achieved a passwordless identity system. Learn their approach and the consequences of the effort.

### PR DESCRIPTION
**Pull request by heyawhite**
Fixes #7392 

Changes proposed in this pull request:

- New case study for Yahoo! JAPAN

Awaiting final confirmation from partners and @agektmr, after the [Golden Week](https://en.wikipedia.org/wiki/Golden_Week_(Japan)). PR opened for preview.

### CODE REVIEW

File: src/site/content/en/blog/yahoo-japan-identity-case-study/index.md [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+    Learn about how Yahoo! Japan built a passwordless identity system.
+description: |
+    Yahoo! JAPAN achieved a passwordless identity system. Learn their approach and the consequences of the effort.
+authors:
+  - yuyaito
+  - agektmr
+  - alexandrawhite
+date: 2022-05-10
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/tqK2Tb4rUrxheagyYbDj.png
+alt:
```
Comment (rachelandrew) [inline src/site/content/en/blog/yahoo-japan-identity-case-study/index.md:14]: this is why presubmit is failing.

Comment (heyawhite) [reply src/site/content/en/blog/yahoo-japan-identity-case-study/index.md:14]: Oh sure, fun! Waiting for my update to push.

Comment (heyawhite) [reply src/site/content/en/blog/yahoo-japan-identity-case-study/index.md]: The preview may still be failing -- any insights as to why?

**Comment by agektmr** [comment]
I'm still asking Yuya for a final comment on SMS matter. As soon as I get the answer and accommodate it if necessary, I'll give it LGTM.

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
Can you please hold on publishing for a moment?

File: src/site/content/en/blog/yahoo-japan-identity-case-study/index.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+2.  [FIDO with WebAuthn](https://developers.google.com/identity/fido)
+
+In addition, we also offer authentication methods such as e-mail
+authentication, password combined with SMS OTP (one time password), and
+password combined with email OTP.
+
+{% Aside 'important' %}
+In general, phone numbers are known to be recycled and can be hijacked with
+social engineering. Yahoo! JAPAN restricts their service to phone carriers
+operating inside Japan and prohibits VoIP SMS.
```
Comment (agektmr) [inline src/site/content/en/blog/yahoo-japan-identity-case-study/index.md:108]: After making this change, it's ready to be published.

```suggestion
Yahoo! JAPAN restricts their service to phone carriers
operating inside Japan and prohibits VoIP SMS.
```

**Pull request merged**